src/utils.ts

```diff
@@ -104,11 +104,17 @@ export function getEnv(prefix = ''): (name: string) => string | undefined {
     envsLowerCase[envKey.toLowerCase()] = process.env[envKey];
   }
 
-  return (name: string) => {
+  return (name: string): string | undefined => {
     return (
       envsLowerCase[`${prefix}${name}`.toLowerCase()] ||
       envsLowerCase[name.toLowerCase()] ||
       undefined
     );
   };
 }
+
+export function setEnv(key: string, value: string | undefined): void {
+  if (value) {
```

Comment:
I'm going to assume that we're going to ignore [all falsey values in JavaScript](https://stackoverflow.com/a/19839953)?